src/kibana/netmon_libs/custom_modules/save_rule/services/searchAuditor.js

```diff
@@ -1,94 +1,90 @@
 define(function (require) {
-    var app = require('modules').get('app/dashboard');
-    var _ = require('lodash');
-    var moment = require('moment');
-    var angular = require('angular');
+   var app = require('modules').get('app/dashboard');
+   var _ = require('lodash');
+   var moment = require('moment');
+   var angular = require('angular');
+   var timefilter = require('components/timefilter/timefilter');
 
-    var timefilter = require('components/timefilter/timefilter');
-    app.factory('searchAuditor', function (elasticSearchFields,
-      Restangular, timefilter) {
-         var FORMAT_STRING = 'YYYY/MM/DD HH:mm:ss';
-         var INDEX_PATTERN = '[network_]YYYY_MM_DD';
+   app.factory('searchAuditor', function (elasticSearchFields, Restangular, timefilter, $http) {
+      var FORMAT_STRING = 'YYYY/MM/DD HH:mm:ss';
```

Comment:
please make these consts (I know... not your fault!)

Reply:
Np, done!